inf-clojure.el

```diff
@@ -64,7 +64,8 @@
 ;;; Code:
 
 (require 'comint)
-(require 'clojure-mode)
+(require 'clojure-mode nil 'no-error)
```

Comment:
I think the mode still has a hard dependency on `clojure-mode`, as it uses functions from it internally. That's why you can't really make this optional.

Reply:
(all the functions that start with `clojure-` are coming from there)

Reply:
Oops. Makes sense. I just reverted this line to just `(require 'clojure-mode)`